stress_testing/functions.js

```diff
@@ -0,0 +1,25 @@
+const { generatefFeaturesTableRecord } = require('./data_generator.js');
```

Comment:
Is that a typo, or is the lowercase f at the end of generate intentional?

Reply:
Totally a typo smh, good catch! Fixing now